stable/graphite/templates/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     spec:
       initContainers:
       - name: create-syncdb
-        image: {{ .Values.image.repository }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
```

Comment:
If you already have tested the 1.5.x version with this chart the image version should be also raised in values.yaml and README...